contracts/upgradeable/mocks/ERC721MockUpgradeable.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.18;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+
+/**
+ * @title ERC721MockUpgradeable
+ * Used for tests with non RMRK implementer
+ */
+contract ERC721MockUpgradeable is ERC721Upgradeable {
```

Comment:
I do not think we need this. Unless it is inherited by another mock we can test with regular ERC721.